docs/api/views/Views.md

```diff
@@ -6,9 +6,9 @@ Navigation views are controlled React components that can present the current na
 
 ## Built in Views
 
-- [CardStack](https://github.com/react-community/react-navigation/blob/master/src/views/CardStack.js) - Present a stack that looks suitable on any platform
-    + [Card](https://github.com/react-community/react-navigation/blob/master/src/views/Card.js) - Present one card from the card stack, with gestures
-    + [Header](https://github.com/react-community/react-navigation/blob/master/src/views/Header.js) - The header view for the card stack
+- [CardStack](https://github.com/react-community/react-navigation/blob/master/src/views/CardStack) - Present a stack that looks suitable on any platform
```

Comment:
I believe it was the initial intention of these links to redirect to the `.js` files, not their directories. If you can make sure these links direct to the correct `CardStack.js` et al.